StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/CardSectionWithScanner/CardSectionWithScannerElement.swift

```diff
@@ -32,14 +32,31 @@ final class CardSection: ContainerElement {
     let cardSection: SectionElement
 
     // References to the underlying TextFieldElements
+    let nameElement: TextFieldElement?
     let panElement: TextFieldElement
     let cvcElement: TextFieldElement
     let expiryElement: TextFieldElement
     let theme: ElementsUITheme
 
-    init(theme: ElementsUITheme = .default) {
+    init(
+        collectName: Bool = false,
+        defaultName: String? = nil,
+        theme: ElementsUITheme = .default
+    ) {
         self.theme = theme
-        let panElement = PaymentMethodElementWrapper(TextFieldElement.PANConfiguration(), theme: theme) {  field, params in
+        let nameElement = collectName
+            ? PaymentMethodElementWrapper(
+                TextFieldElement.NameConfiguration(
+                    type: .full,
+                    defaultValue: defaultName,
+                    label: STPLocalizedString("Name on card", "Label for name on card field")),
```

Comment:
I found this surprising because I would have expected name to be a part of the billing section, not showing up as "name on card", but i suspect there is good reason for this.  Any documentation you can point me to on this?

Reply:
There isn't a good reason for this, other than it's what I think looks best, I experimented with multiple ways and showed them to @jarvis-stripe, for context, checkout in web shows the name field on its own and it looks kinda weird, but it is one of the options.

This is web checkout:

<img width="370" alt="image (1)" src="https://user-images.githubusercontent.com/114543737/221298574-1f687e06-4084-41c6-8e82-a8e38a106cab.png">

And these are some of the options I tried:
![image (2)](https://user-images.githubusercontent.com/114543737/221298693-598283a0-2677-4cc4-875e-73fc0b735c24.png)
![image (3)](https://user-images.githubusercontent.com/114543737/221298694-fc716225-b147-47b4-8133-1a5ede42bd6d.png)
![image (4)](https://user-images.githubusercontent.com/114543737/221298697-340ff7f8-b667-481e-ab16-422063f47769.png)